arrow/src/main/java/org/apache/iceberg/arrow/vectorized/parquet/VectorizedParquetDefinitionLevelReader.java

```diff
@@ -193,7 +193,7 @@ public void readBatchOfDictionaryEncodedLongs(
         case RLE:
           if (currentValue == maxDefLevel) {
             dictionaryEncodedValuesReader.readBatchOfDictionaryEncodedLongs(vector,
-                idx, numValues, dict, nullabilityHolder);
+                idx, numValues, dict, nullabilityHolder, typeWidth);
```

Comment:
What about the call in `case PACKED` just below? Does that also need to use the `typeWidth`?

Reply:
Good catch, looks like I missed it. Will fix here and other places.